src/Chainweb/BlockHeader.hs

```diff
@@ -374,16 +378,40 @@ epochStart ph@(ParentHeader p) adj (BlockCreationTime bt)
     --    b) adjusting timeBlocked in both directions, possibly based off the
     --       avg?
     --
-    timeBlocked = maximum adjCreationTimes .-. _blockCreationTime p
+    _adjustmentMax = maximum adjCreationTimes .-. _blockCreationTime p
```

Comment:
this looks unused.

Reply:
yes, this is work in progress. It's not yet decided what will be the final algorithm here. So, I kept the the alternative to be able to switch quickly.